razorpay/razorpay.php

```diff
@@ -200,11 +200,14 @@ public function hookHeader()
 
             $cart_presenter = new CartPresenter();
 
+            $amount = ($this->context->cart->getOrderTotal() * 100);
```

Comment:
Can you host this somewhere so we can test it out?

Reply:
Let me try it.

Reply:
Looks like, we need some staging environment to test the changes thoroughly.